generator/src/main/java/com/linkedin/pegasus/generator/PegasusDataTemplateGenerator.java

```diff
@@ -49,6 +44,7 @@
    * The system property that specifies whether to generate classes for externally resolved schemas
    */
   public static final String GENERATOR_GENERATE_IMPORTED = "generator.generate.imported";
+  public static final String GENERATOR_GENERATE_PATH_LOWERCASE = "generator.generate.path.lowercase";
```

Comment:
rename for consistency:
PATH_LOWERCASE -> LOWERCASE_PATH

Reply:
My original thought was that having grouping of "path" with leaf node "lowercase" made more sense than having a "lowercase" grouping with "path" as a leaf node. 

I will change if this didn't  logic didn't make sense to you.